novawallet/en.lproj/Localizable.strings

```diff
@@ -1762,3 +1762,8 @@
 "common.use.max.due.fee.message" = "You can use up to %@ since you need to pay %@ for network fee.";
 "common.pay.anywhere" = "Pay anywhere";
 "common.nova.card" = "Nova Card";
+"common.estimated.timer" = "Estimated ~%@";
+"card.open.pending.sheet.title" = "Your card is being issued!";
+"card.open.pending.sheet.message" = "It can take up to 5 minutes.\nThis window will be closed automatically.";
```

Comment:
Let's parametrise by the value of time we have in ```PayCardPresenter``` instead of hardcoding 5 minutes